src/app.py

```
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)